cloud-azure/src/main/java/com/sequenceiq/cloudbreak/cloud/azure/service/AzureResourceNameService.java

```diff
@@ -43,20 +41,20 @@ public String resourceName(ResourceType resourceType, Object... parts) {
 
     private String volumeSetResourceName(Object[] parts) {
         checkArgs(VOLUMESET_PART_COUNT, parts);
+        String hash = String.valueOf(parts[VOLUMESET_PART_COUNT - 1]);
         String name = instanceName(parts);
-        name = trimHash(name);
-        name = appendHash(name, new Date());
+        name = appendHash(name, hash);
         name = adjustBaseLength(name, maxResourceNameLength);
         return name;
     }
 
     private String attachedDiskResourceName(Object[] parts) {
         checkArgs(ATTACHED_DISKS_PART_COUNT, parts);
-        String cnt = String.valueOf(parts[ATTACHED_DISKS_PART_COUNT - 1]);
+        String cnt = String.valueOf(parts[ATTACHED_DISKS_PART_COUNT - 2]);
```

Comment:
We should log all the generated resource names here.

Reply:
The resourceName() method is extended with the logging.